diff --git a/bin/roave-backward-compatibility-check.php b/bin/roave-backward-compatibility-check.php
index 8909d19..c92ba23 100644
--- a/bin/roave-backward-compatibility-check.php
+++ b/bin/roave-backward-compatibility-check.php
@@ -53,6 +53,14 @@ use function file_exists;
         throw new RuntimeException('Could not find Composer autoload.php');
     })();

+    //PATCH START
+    //These files are used to test the plugin. These are normally provided by the application that uses the plugin, but
+    //these two are used to run the automated tests. Without these lines, the bc check fails saying that the files
+    //cannot be found, since the non-test autoloader does not include the tests/Application/src directory.
+    require_once __DIR__ . '/../../../../tests/Application/src/Entity/Addressing/Address.php';
+    require_once __DIR__ . '/../../../../tests/Application/src/Entity/Order/Order.php';
+    //PATCH END
+
     $application = new Application('roave/backward-compatibility-check', Versions::getVersion('roave/backward-compatibility-check'));
     $helperSet   = $application->getHelperSet();
     $input       = new ArgvInput();
