Index: vendor/sylius/sylius/src/Sylius/Behat/Page/Shop/Checkout/AddressPage.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/vendor/sylius/sylius/src/Sylius/Behat/Page/Shop/Checkout/AddressPage.php b/vendor/sylius/sylius/src/Sylius/Behat/Page/Shop/Checkout/AddressPage.php
--- a/vendor/sylius/sylius/src/Sylius/Behat/Page/Shop/Checkout/AddressPage.php
+++ b/vendor/sylius/sylius/src/Sylius/Behat/Page/Shop/Checkout/AddressPage.php	(date 1768408224673)
@@ -249,14 +249,14 @@
 
     public function selectShippingAddressFromAddressBook(AddressInterface $address): void
     {
-        $this->getElement('shipping_address_book')->selectOption($address->getId());
+        $this->getElement('shipping_address_book')->selectOption((string) $address->getId());
 
         $this->waitForElementUpdate('form');
     }
 
     public function selectBillingAddressFromAddressBook(AddressInterface $address): void
     {
-        $this->getElement('billing_address_book')->selectOption($address->getId());
+        $this->getElement('billing_address_book')->selectOption((string) $address->getId());
 
         $this->waitForElementUpdate('form');
     }
